### PR TITLE
Test

### PR DESCRIPTION
**Pull request by benslamahafedh**

### CODE REVIEW

**Comment by benslamahafedh** [comment]
[DB-connection](https://trello.com/c/nVP7Dgz3/4-db-connection)

File: database_connection.js [hunk 1]
```diff
@@ -1,31 +1,23 @@
 const mysql = require("mysql2/promise");
 
-const query = async ()=>{
+
+
+const main = async (db_request)=>{
```
Comment (benslamahassen) [inline database_connection.js]: You need to work on your variable names 😂 so `db_request` can just be `request` or even `req`

File: database_connection.js [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
             database: "tasks",
-            
         });
-        const query = async (query)=>{
-            await connection.query(query, (err,res) => {
-                if (err) throw res.status(404).json(err);
-                
-                });
-        };
+        const result= await connection.execute(db_request)
```
Comment (benslamahassen) [inline database_connection.js]: `result` is usually `res`

File: index.js [hunk 3]
```diff
@@ -1,96 +1,73 @@
 const services = require("./services/servicees");
-const Joi = require("joi");
 const express = require("express");
 const app = express();
 app.use(express.json());
+app.set('view engine','pug');
 
-// db.query (`TRUNCATE TABLE tasks;`,'delete')
+const response=services.getall();
```
Comment (benslamahassen) [inline index.js]: getAll() mch getall() hethi tetsema camelCase notation w heya best practice fil JS w node

**Review by benslamahassen — COMMENTED** [review]
Also never commit commented code

File: services/servicees.js [hunk 4]
```diff
@@ -1,33 +1,64 @@
...
-    db.query("SELECT * FROM tasks;");
+    try{
+        app.get('/',(req,res)=>{
+            const response = db.main("SELECT * FROM tasks;");
+            res.status(200).json(response)
+        })
+    }catch(err){
+        console.error(err);
+    };
 };
```
Comment (benslamahafedh) [inline services/servicees.js:24]: l condition matemchich ki nda5el id mch mawjoud check it out

Comment (benslamahassen) [reply services/servicees.js:24]: Any logs ?

Comment (benslamahafedh) [reply services/servicees.js:24]: chma3netha

Comment (benslamahassen) [reply services/servicees.js:24]: Ma3neha matal3lekch error fel terminal wela heja ?

Comment (benslamahafedh) [reply services/servicees.js:24]: la la

Comment (benslamahassen) [reply services/servicees.js:24]: Mela kifeh te5demch ? Mafhemtech

Comment (benslamahafedh) [reply services/servicees.js:24]: /tasks/id supposé tjiblik task by id but if il id mch fil db supposé ta3tina status 204 w errors msg '204 no content" ki njarab b postman yraja3li empty list w status 200

Comment (benslamahassen) [reply services/servicees.js:24]: Ayh bien sur lazmek ta3mel el error enti wa7dek 
If ( result fer8a ) return res.status...

Comment (benslamahafedh) [reply services/servicees.js:24]: that is what line 30 is about illi 3morha la da5let lil condition heki ahat ki na3ti id mch mawjoud

**Pull request merged**